core/src/main/java/org/infinispan/stats/impl/CacheContainerStatsImpl.java

```diff
@@ -64,7 +64,11 @@ public void resetStatistics() {
       if (getStatisticsEnabled()) {
          for (String cn : cm.getCacheNames()) {
             if (cm.cacheExists(cn)) {
-               cm.getCache(cn).getAdvancedCache().getStats().reset();
+               try {
+                  cm.getCache(cn).getAdvancedCache().getStats().reset();
+               } catch (SecurityException e) {
+                  // no-op, skipping this cache
```

Comment:
I was thinking that we could use a proper user to call `CacheContainerStatsImpl.resetStatistics()`, and this method shouldn't have to worry about security at all.

I don't really like the idea of skipping caches we don't have access to, because the user gets no feedback why he's seeing partial statistics. @tristantarrant WDYT?